Blog Application/index.html

```diff
@@ -0,0 +1,112 @@
+		<div class="modal-dialog">
+			<div class="modal-content">
+				<div class="modal-header">
+					<h5 class="modal-title" id="staticBackdropLabel">Add New Task</h5>
+					<button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+				</div>
+				<div class="modal-body">
+					<form>
+						<div class="mb-3">
+							<label for="imageurl" class="form-label">Image URL</label>
+							<input type="url" class="form-control" id="imageurl" aria-describedby="emailHelp" placeholder="https://images.hello.com/hello.png">
+						</div>
+						<div class="mb-3">
+							<label for="title" class="form-label">Title</label>
+							<input type="text" class="form-control" id="title" placeholder="Artificial Intelligence">
+						</div>
+						<div class="mb-3">
+							<label for="type" class="form-label">Type</label>
+							<input type="text" class="form-control" id="type" placeholder="AI">
+						</div>
+						<div class="mb-3">
+							<label for="description" class="form-label">Description</label>
+							<textarea rows="4" class="form-control" id="description"></textarea>
+						</div>
+					</form>
+				</div>
+				<div class="modal-footer">
+					<button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
+					<button type="button" class="btn btn-primary" data-bs-dismiss="modal" onclick="saveChanges()">Save Changes</button>
+				</div>
+			</div>
+		</div>
+	</div>
+	<!--  card modal -->
+	<div class="modal fade" id="showblog" tabindex="-1" aria-labelledby="showTaskLabel" aria-hidden="true">
+		<div class="modal-dialog modal-lg">
+			<div class="modal-content">
+				<div class="modal-body blog__modal__body"></div>
+				<div class="modal-footer">
+					<button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
+				</div>
+			</div>
+		</div>
+	</div>
+	</div>
+	<!-- navbar -->
+	<nav class="navbar navbar-expand-md navbar-light bg-light shadow-sm">
+		<div class="container-fluid"> <a class="navbar-brand  fw-bold text-primary" href="#">Blog</a>
+			<button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation"> <span class="navbar-toggler-icon"></span> 
+			</button>
+			<div class="collapse navbar-collapse" id="navbarSupportedContent">
+				<ul class="navbar-nav me-auto mb-2 mb-lg-0">
+					<li class="nav-item"> <a class="nav-link active" aria-current="page" href="#">Home</a> 
+					</li>
+				</ul>
+				<button type="button" class="btn btn-primary rounded-pill" data-bs-toggle="modal" data-bs-target="#staticBackdrop"><i class="fas fa-plus"></i> Add New</button>
+			</div>
+		</div>
+	</nav>
+	<div class="container">
+		<section>
+			<div class="row blog__container mt-5 mb-3 ">
+				<!-- <div class="col-lg-4 col-md-6">
```

Comment:
Is the comment required? If not then I think you can remove it. The rest looks really good. Good work @khushi-purwar !

Reply:
No, it is not required. I will remove it.